app/assets/stylesheets/admin/views/_attachments.scss

```diff
@@ -1,6 +1,7 @@
 .right-to-left {
   input,
-  textarea {
+  textarea,
+  .govspeak {
     direction: rtl;
   }
```

Comment:
Here is the documentation explaining our use of BEM https://github.com/alphagov/govuk_publishing_components/blob/main/docs/component_conventions.md#bem

I agree with Kevin, we should use class names rather than tag name. It's part of the convention plus it's protects against future changes.

As we're using components, you might need to target the specific class for the input within the component with a wrapping class.